src/main.rs

```diff
@@ -31,14 +38,26 @@ fn main() {
 
 pub struct MyGame {
     // Your state here...
+    player1: Player,
```

Comment:
Why not add a struct that encapsulates all player data and then give `MyGame` a Vec/array of said type?

Reply:
This was originally gonna be just one _Player_ I just dropped the second dude in there to see what it looked like. This is essentially what I ended up doing later.